package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "4.5.3",
+  "version": "4.5.6",
```

Comment:
This is a big jump. Missing rebase?

Reply:
Nope, the package*.json files with 4.5.3 file was committed previously, but because this PR wasn't committed it fell behind.  When I rebased, the package*.json files were in conflict.  This commit updates the app version.  I believe you've already approved this PR.

Reply:
When I rebase and have a package file conflict, I fix the conflict manually. This usually means taking the incoming change (from the rebase) and then updating the app version again (npm i). Maybe Ketaki's process of updating the app version LAST is a good idea :)

I did approve previously but I cannot see the actual new changes due to the changes from a previous PR displaying a lot of diffs.